src/components/Card.js

```diff
@@ -117,11 +129,15 @@ Card.propTypes = {
   category: PropTypes.string.isRequired,
   body: PropTypes.string.isRequired,
   backgroundImage: PropTypes.string.isRequired,
-  link: PropTypes.number.isRequired,
+  link: PropTypes.number.isRequired
 };
 
 CardList.propTypes = {
   article: PropTypes.array.isRequired,
   title: PropTypes.string.isRequired,
   className: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/require-default-props
+  pageLimit: PropTypes.number,
+  // eslint-disable-next-line react/require-default-props
```

Comment:
Same here, cant the default function be one that does nothing?

Reply:
@KvNGCzA feedback well received.
@KvNGCzA done.